frontends/web/src/routes/account/summary/chart.module.css

```diff
@@ -123,6 +123,7 @@
 .totalUnit {
   color: var(--color-secondary);
   display: inline-block;
+  margin-bottom: 5px;
```

Comment:
I'm not sure if this is same on all OS.. but on my machine 3px would align the font baseline perfectly

![Screen Shot 2024-03-19 at 15 03 04](https://github.com/digitalbitbox/bitbox-wallet-app/assets/546900/01ad6fb6-dd0f-47a9-ab9e-86864722c95c)

Reply:
thank you 🙏 

that was it looks consistent with the formatting in account overview